app/views/repository/_card_badge.html.erb

```diff
@@ -1,3 +1,8 @@
 <div class="card-badge">
-  <%= image_tag "tech-#{technology ||= 'default'}.svg", alt: "Card of the repository #{technology ||= 'default'}" %>
+  <img
+    src="<%= '/assets/' + technology_image_pattern + '.svg' %>"
+    alt="<%= 'Card of the repository ' + technology %>"
+    width="23"
+    onerror="this.onerror=null; this.src='/assets/default.svg'; this.alt='Card of the repository default'"
+  >
```

Comment:
algum motivo especifico para não utilizar o metodo `image_tag` aqui? Ele é bem importante pois ja vem com algumas checagens de segurança e de maneira geral facilita nossa vida por aceitar codigo ruby:

```suggestion
  <%= image_tag "/assets/#{technology_image_pattern}.svg", alt: "Card of the repository #{technology}", size: '23', onerror: "this.onerror=null; this.src='/assets/default.svg'; this.alt='Card of the repository default'" %>
```

Aqui eu to assumindo que estamos utilizando o metodo helper (com o mesmo nome da propriedade) como comentado em alguns comentarios acima.

Reply:
Existe sim, tentei usar esse método mas se uma imagem não for encontrada, ele lança um erro, ao invés de cair no `onerror` que é a melhor forma que eu achei de definir a imagem default para o caso de não encontrar uma. É meio gambi, mas a outra forma seria verificar o nome dos arquivos e se não achasse o padrão, significaria que a imagem não existe, aí usaria a imagem default.

Reply:
hum entendi, bom acho que podemos seguir com isso hoje e dps checamos se existe alguma forma melhor de se fazer

Reply:
Pois é, a questão toda é que a gente precisa identificar se existe ou não imagem para aquela tech.